PlottingScripts/Basic_Plot.py

```diff
@@ -0,0 +1,128 @@
+    small_angle = big_angle / (minvalues.size+2)
+
+    p = figure(plot_width=width, plot_height=height, title="",
+               x_axis_type=None, y_axis_type=None,
+               x_range=(-420, 420), y_range=(-420, 420),
+               min_border=0, outline_line_color="black",
+               background_fill="#f0e1d2", border_fill="#f0e1d2")
+
+    p.xgrid.grid_line_color = None
+    p.ygrid.grid_line_color = None
+
+    # annular wedges divided into smaller sections for bars
+    angles = np.pi/2 - big_angle/2 - df.index.to_series()*big_angle
+    # circular axes and labels
+    labels = np.power(10.0, np.arange(0, -5, -1))
+    labz = np.power(10.0, np.arange(0, -4.5, -0.5))
+    # Re-size if no values are above 0.1
+    if maxval < 0.1:
+        labels = np.delete(labels, 0)
+    # Size of radii of each label
+    pdb.set_trace()
+    rad = (((np.log10(labz / labels[0])) + labels.size) *
+           (outer_radius - inner_radius) / labels.size + inner_radius)
+    radii = (((np.log10(labels / labels[0])) + labels.size) *
+             (outer_radius - inner_radius) / labels.size + inner_radius)
+    # Add zero label to labels and radii
+    labels = np.append(labels, 0.0)
+    radii = np.append(radii, inner_radius)
+    rad = np.append(rad, inner_radius)
+    p.circle(0, 0, radius=rad, fill_color=None, line_color="white")
+    p.text(0, radii[:], [str(r) for r in labels[:]],
+           text_font_size="8pt", text_align="center", text_baseline="middle")
+    # Plot the values of each stat for each parameter.
+    for c in range(0, Cols.size):
+        p.annular_wedge(0, 0, inner_radius, (((np.log10(df[Cols[c]] /
+                                              labels[0])) + labels.size) *
+                                             (outer_radius - inner_radius) /
+                                             labels.size + inner_radius),
+                        -big_angle + angles + (2*c + 1)*small_angle,
+                        -big_angle + angles + (2*c + 2)*small_angle,
+                        color=stat_color[c])
+    # radial axes
+    p.annular_wedge(0, 0, inner_radius-10, outer_radius+10,
+                    -big_angle+angles, -big_angle+angles, color="black")
+
+    # Placement of parameter labels
+    xr = radii[0]*np.cos(np.array(-big_angle/2 + angles))
+    yr = radii[0]*np.sin(np.array(-big_angle/2 + angles))
+
+    label_angle = np.array(-big_angle/2+angles)
+    label_angle[label_angle < -np.pi/2] += np.pi
+    # Placing Labels and Legend
+    p.text(xr, yr, df.Parameter, angle=label_angle,
+           text_font_size="9pt", text_align="center", text_baseline="middle")
+
+    p.rect([-40, -40, -40], [18, 0, -18], width=30, height=13,
+           color=list(stat_color.values()))
+    p.text([-15, -15, -15], [18, 0, -18], text=Cols,
+           text_font_size="9pt", text_align="left", text_baseline="middle")
+
+    output_file("Sensitivity.html", title=Dataframe)
```

Comment:
Two points on the output file:
-  I think we should save plots outside the repository (perhaps next to the data directory?)
-  Can you make another parameter for the function that indicates whether or not a plot should be saved?  Often a user will want to explore a bunch of plots in an interactive session, but doesn't want to save them all until they get exactly the plot they would like.